### ISSUE TITLE
Incompatibility with DunglasActionBundle

### ISSUE THREAD

**Issue by kix** [fesor/request-objects#10]
Looks like the bundle is incompatible with DunglasActionBundle and, consequently, API Platform, which uses some of DunglasActionBundle's features.
```
(1/1) FatalThrowableErrorType error: ReflectionFunction::__construct() expects parameter 1 to be string, object given
--
in RequestObjectBinder.php (line 112)
at ReflectionFunction->__construct(object(DocumentationAction))in RequestObjectBinder.php (line 112)
at RequestObjectBinder->matchActionArguments(object(DocumentationAction))in RequestObjectBinder.php (line 51)
at RequestObjectBinder->bind(object(Request), object(DocumentationAction))in RequestObjectEventListener.php (line 26)
at RequestObjectEventListener->onKernelController(object(FilterControllerEvent), 'kernel.controller', object(TraceableEventDispatcher))
```

**Comment by fesor** [comment]
Will check this in a few days. Thanks

**Comment by ghost** [comment]
I have same issue, but with NelmioApiDocBundle.

In both bundles the routes required identically:

 NelmioApiDocBundle:
>    defaults: { _controller: nelmio_api_doc.controller.swagger_ui }
    path:   /


 DunglasActionBundle
>  foo:
    path:      /foo/{bar}
    defaults:  { _controller: 'AppBundle\Action\Homepage' } 

and in matchActionArguments method of Fesor\RequestObject $action is object.
But this case not catched in code.

**Quick dirty solution:**
change
> defaults: { _controller: nelmio_api_doc.controller.swagger_ui }

to 

> defaults: { _controller: nelmio_api_doc.controller.swagger_ui:__invoke }

**Comment by fesor** [comment]
Fixed in v0.3.0.
